src/utilities/include/par.h

```diff
@@ -84,9 +79,7 @@ inline constexpr ExecutionPolicy autoPolicy(size_t size) {
     return thrust::NAME(thrust::cpp::par, args...);                 \
   }
 
-#if MANIFOLD_PAR != 'T' || \
-    (TBB_INTERFACE_VERSION >= 10000 && __has_include(<pstl/glue_execution_defs.h>))
-#if MANIFOLD_PAR == 'T'
+#if MANIFOLD_PAR == 'T' && __has_include(<pstl/glue_execution_defs.h>)
```

Comment:
It looks like with LLVM on my mac, I'm failing this pstl check, which is why I commented out the `std::copy_if(std::execution::par_unseq` stuff below. Is there a more generic way to check for the presence of PSTL?